packages/api-explorer/src/components/ExploreType/ExploreProperty.tsx

```diff
@@ -166,17 +177,17 @@ export const ExplorePropertyDetail: FC<ExplorePropertyProps> = ({
   property,
 }) => (
   <Space style={{ fontSize: 'small', marginLeft: '10rem' }}>
-    <FlexItem width="10rem">
+    <Box width="10rem">
```

Comment:
Maybe some day I'll know when to use `Box` vs. `FlexItem` but probably not.

Reply:
Generally within a `Space` any element will "just work". I have a personal "beef" with FlexItem since it's really just a funny-named `Box`

Reply:
I see how you like to flex on `FlexItem`. That's ok.